package.json

```diff
@@ -61,7 +61,7 @@
     "@emotion/styled": "^11.8.1",
     "@ethersphere/bee-js": "^5.0.0",
     "@ethersphere/swarm-extension": "^0.7.0",
-    "@fairdatasociety/fdp-storage": "https://github.com/fairDataSociety/fdp-storage.git#fix/bee-js-762",
+    "@fairdatasociety/fdp-storage": "github:fairDataSociety/fdp-storage#fix/bee-js-new-762",
```

Comment:
why does it refer a different fdp-storage version than the one that the library uses? do these two different libraries share the same interfaces and logic? if so, why is it necessary? does the 0.7 fdp-storage version not work with web-workers?

Reply:
Now the `bee-js-new-762` branch contains latest version. I just updated types for the js-library to be compatible with this version.